tests/unit/sagemaker/jumpstart/test_types.py

```diff
@@ -90,14 +69,14 @@ def test_jumpstart_model_specs():
     assert specs1.hosting_ecr_specs == JumpStartECRSpecs(
         {
             "framework": "pytorch",
-            "framework_version": "1.7.0",
+            "framework_version": "1.5.0",
             "py_version": "py3",
         }
     )
     assert specs1.training_ecr_specs == JumpStartECRSpecs(
         {
             "framework": "pytorch",
-            "framework_version": "1.9.0",
+            "framework_version": "1.5.0",
```

Comment:
just curious: why these changes? Are you weakening the test here?

Reply:
I changed the base model spec